kani-compiler/src/codegen_cprover_gotoc/codegen/rvalue.rs

```diff
@@ -419,6 +419,7 @@ impl<'tcx> GotocCtx<'tcx> {
 
     pub fn codegen_rvalue(&mut self, rv: &Rvalue<'tcx>, loc: Location) -> Expr {
         let res_ty = self.rvalue_ty(rv);
+        debug!(?rv, "codegen_rvalue");
```

Comment:
When should `trace!(...)` be used instead of `debug!(...)`? Or are some of these macro calls leftovers?

Reply:
We don't have a clear guideline yet but trace!() statements get removed from release builds so I tend to use them to provide more details to the stages of codegen.